specification/network/resource-manager/Microsoft.Network/stable/2024-10-01/virtualNetworkGateway.json

```diff
@@ -5034,7 +5034,7 @@
```

Comment:
`"deprecated": true`
?

Reply:
Added Deprecated: in description, following same pattern as other deprecated properties.